docs/knowledgebase/advanced/block-import.md

```diff
@@ -41,36 +39,34 @@ the block is signed by the appropriate authority.
 
 When the import queue is ready to import a block, it passes the block in question to a method
 provided by the
-[`BlockImport` trait](https://substrate.dev/rustdocs/v2.0.0-rc4/sp_consensus/block_import/trait.BlockImport.html).
+[`BlockImport` trait](https://substrate.dev/rustdocs/v2.0.0-rc5/sp_consensus/block_import/trait.BlockImport.html).
 This `BlockImport` trait provides the behavior of importing a block into the node's local state
 database.
 
-One implementor of the `BlockImport` trait that is used in every Substrate node is the
-[`Client`](https://substrate.dev/rustdocs/v2.0.0-rc4/sc_service/client/index.html), which contains the node's entire
+One implementer of the `BlockImport` trait that is used in every Substrate node is the
+[`Client`](https://substrate.dev/rustdocs/v2.0.0-rc5/sc_service/client/index.html), which contains the node's entire
 block database. When a block is imported into the client, it is added to the main database of blocks
 that the node knows about.
 
 ## The Block Import Pipeline
 
-In the simplest cases, blocks are imported directly into the client. But most consensus engines will
+In the simplest mode, blocks are imported directly into the client. However, most consensus engines will
 need to perform additional verification on incoming blocks, update their own local auxiliary
 databases, or both. To allow consensus engines this opportunity, it is common to wrap the client in
 another struct that also implements `BlockImport`. This nesting leads to the term "block import
 pipeline".
 
 An example of this wrapping is the
-[`PowBlockImport`](https://substrate.dev/rustdocs/v2.0.0-rc4/sc_consensus_pow/struct.PowBlockImport.html), which
-holds a reference to another type that also implements `BlockImport`. This allows the PoW consensus
-engine to do its own import-related bookkeeping and then pass the block to the nested `BlockImport`,
-probably the client. This pattern is also demonstrated in
-[`AuraBlockImport`](https://substrate.dev/rustdocs/v2.0.0-rc4/sc_consensus_aura/struct.AuraBlockImport.html),
-[`BabeBlockImport`](https://substrate.dev/rustdocs/v2.0.0-rc4/sc_consensus_babe/struct.BabeBlockImport.html), and
-[`GrandpaBlockImport`](https://substrate.dev/rustdocs/v2.0.0-rc4/sc_finality_grandpa/struct.GrandpaBlockImport.html).
-
-`BlockImport` nesting need not be limited to one level. In fact, it is common for nodes that use
-both an authoring engine and a finality gadget to layer the nesting even more deeply. For example,
-Polkadot's block import pipeline consists of a `BabeBlockImport`, which wraps a
-`GrandpaBlockImport`, which wraps the `Client`.
+[`PowBlockImport`](https://substrate.dev/rustdocs/v2.0.0-rc5/sc_consensus_pow/struct.PowBlockImport.html), which
+holds a reference and can implement `BlockImport`. Thus allowing the PoW consensus
```

Comment:
I'm not sure that "which holds a reference and can implement `BlockImport`" makes sense.

The point of this text is that you can daisy chain these `BlockImports` together. `PowBlockImport`, demonstrates this because its `inner` fiels must also be `BlockImport`. Check out the trait bounds on `I` in this screenshot from the `PowBlockImport` docs.

![image](https://user-images.githubusercontent.com/2915325/89419692-ac10ad00-d6ff-11ea-8b9d-c455d6d852bb.png)

Reply:
thanks you're right @JoshOrndorff I'll rework this